gradle.properties

```diff
@@ -1,5 +1,5 @@
 GROUP=com.alecstrong
-VERSION_NAME=0.4.0-SNAPSHOT
+VERSION_NAME=0.3.16
```

Comment:
Or should I use `-SNAPSHOT` and you do the releasing stuff?

Reply:
yea just snapshot and I'll release